### PR TITLE
IW-1421 | fix submitting search query

### PR DESCRIPTION
**Pull request by mszabo-wikia**
When user submits search form using Enter key, they should be redirected to the search results page.

https://wikia-inc.atlassian.net/browse/IW-1421

### CODE REVIEW

File: tests/integration/components/global-navigation/search-test.js [hunk 1]
```diff
@@ -79,27 +79,26 @@ module('Integration | Component | global-navigation/search', function (hooks) {
...
-	// 		onSearchToggleClicked=onSearchToggleClicked
-	// 	}}`);
-
-	// 	const searchInputSelector = '.wds-global-navigation__search-input';
-
-	// 	await click('.wds-global-navigation__search-toggle-icon');
-	// 	await fillIn(searchInputSelector, 'query');
-	// 	await triggerKeyEvent('.wds-global-navigation__search-container', 'keydown', 13);
-	// });
+	test('search input results are shown after click on Enter', async function (assert) {
```
Comment (hakubo) [inline tests/integration/components/global-navigation/search-test.js]: why it started to work?

File: app/components/global-navigation/search.js [hunk 2]
```diff
@@ -355,6 +355,9 @@ export default Component.extend({
 				this.inputField.blur();
 
 				this.closeSearch();
+			} else if (this.goToSearchResults) {
```
Comment (hakubo) [inline app/components/global-navigation/search.js]: it actually works for me on prod and preview. Is the bug on sandbox somewhere?

Comment (mszabo-wikia) [reply app/components/global-navigation/search.js]: The bug is in current dev branch, reproducible if you run it e.g. on devbox

File: tests/integration/components/global-navigation/search-test.js (hunk 1 above)
Comment (mszabo-wikia) [reply tests/integration/components/global-navigation/search-test.js]: it works with fix in this PR. uncommenting it without fix leads to it not working.

**Pull request merged**